src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -2738,7 +2773,7 @@ public ListBoxModel doFillPresetItems(@QueryParameter final boolean useOwnServer
                                               @QueryParameter final String username, @QueryParameter final String password,
                                               @QueryParameter final String timestamp, @QueryParameter final String credentialsId,
                                               @QueryParameter final boolean isProxy, @AncestorInPath Item item) {
-            Jenkins.getInstance().checkPermission(Item.CONFIGURE);
+            item.checkPermission(Item.CONFIGURE);
```

Comment:
```suggestion
            if(item==null){
                FormValidation.ok();
            }else {
                item.checkPermission(Item.CONFIGURE);
            }
```
Please, always check if the context is no null to avoid any error.